examples/regression/4-tutorials/plot_cqr_tutorial.py

```diff
@@ -0,0 +1,411 @@
+"""
+==============================================
+Tutorial for conformalized quantile regression
+==============================================
+
+We will use the sklearn california housing dataset as the
+base for the comparison of the different methods available
+on MAPIE. Two classes will be used:
+:class:`mapie.regression.MapieRegressor` and
+:class:`mapie.quantile_regression.MapieQuantileRegressor`.
+
+The type of conformal methods are chosen with the parameter
```

Comment:
Since it's a tutorial focused on CQR, I would emphasize the use of `MapieQuantileRegressor`:
- Split-conformal is only implemented for now -> you can do it in two ways (split or prefit)
- If prefit, one needs to split your training and calibration sets yourself and train the models
- The base models need to be optimized through quantile regression -> list the models accepted by `MapieQuantileRegressor`

I don't know if this description would be better placed here or before the use of the MAPIE objects, it's up to you.

Reply:
Added this in different parts, the optimization in the `2. Optimizing estimator` and each part in their "respective" part of the notebook. Also, clearly specified that a quantile regressor was needed for CQR.